src/server/graphql/BuildType.js

```diff
@@ -60,7 +60,11 @@ const BuildType = new GraphQLObjectType({
     screenshotDiffs: {
       description: 'Get the diffs for a given build.',
       type: new GraphQLList(ScreenshotDiffType),
-      resolve: build => build.$relatedQuery('screenshotDiffs').orderBy('score', 'desc'),
+      resolve: source => source
+        .$relatedQuery('screenshotDiffs')
+        .innerJoin('screenshots', 'screenshots.id', 'screenshot_diffs.baseScreenshotId')
```

Comment:
Yes, but it can be easily fixed with GraphQL. If the performance is correct, sounds good for me.